src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -1050,6 +1059,17 @@ public function iShouldBeNotifiedThatTheFirstImageShouldHaveAnUniqueCode($imageN
     }
 
     /**
+     * @Then I should be notified that I have to define product variants' prices for newly assigned channels first
+     */
+    public function iShouldBeNotifiedThatIHaveToDefineProductVariantsPricesForNewlyAssignedChannelsFirst()
+    {
+        Assert::same(
+            $this->updateConfigurableProductPage->getValidationMessage('channels'),
```

Comment:
I don't like these methods to, however as they're used in billions of places, they should be refactored all at once... Or maybe we don't care about consistency now and head for splendor, it's also a good way 😄